internal/rootcoord/scheduler.go

```diff
@@ -62,16 +73,33 @@ func (s *scheduler) execute(task task) {
 
 func (s *scheduler) taskLoop() {
 	defer s.wg.Done()
+	ticker := time.NewTicker(Params.ProxyCfg.TimeTickInterval)
+	defer ticker.Stop()
 	for {
 		select {
 		case <-s.ctx.Done():
 			return
+		case <-ticker.C:
+			s.updateLatestTsoAsMinDdlTs()
 		case task := <-s.taskChan:
 			s.execute(task)
 		}
 	}
 }
 
+func (s *scheduler) updateLatestTsoAsMinDdlTs() {
+	if len(s.taskChan) > 0 {
+		return
+	}
+
+	ts, err := s.tsoAllocator.GenerateTSO(1)
+	if err != nil {
+		log.Warn("failed to generate tso", zap.Error(err))
```

Comment:
add more log info, indicate which function  call trigger this warning.

Reply:
OK.